lib/que/adapters/active_record_with_lock.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+# https://github.com/que-rb/que/blob/80d6067861a41766c3adb7e29b230ce93d94c8a4/lib/que/active_job/extensions.rb
+module Que
+    module Adapters
+      class ActiveRecordWithLock < Que::Adapters::ActiveRecord
+        attr_accessor :job_connection_pool, :lock_record
```

Comment:
Don't think we are using the attr_accessor anywhere, can remove the line in that case